README.md

```diff
@@ -32,6 +32,40 @@ Since the project is under active development, we recommend watching the reposit
 regularly for updates. Detailed documentation and usage instructions will be provided as the project
 progresses.
 
+## Testing the Mender MCU Client (POSIX)
+
+### Dependencies
+- cmake
+- libcurl4-openssl-dev
+- libcjson-dev
+- libmbedtls-dev
+
+### Building the Client
+1. Configure the build:
+   ```
+   cmake -B build tests -DCONFIG_MENDER_PLATFORM_FLASH_TYPE="posix" -DCONFIG_MENDER_PLATFORM_LOG_TYPE="posix" -DCONFIG_MENDER_PLATFORM_NET_TYPE="generic/curl" -DCONFIG_MENDER_PLATFORM_SCHEDULER_TYPE="posix" -DCONFIG_MENDER_PLATFORM_STORAGE_TYPE="posix" -DCONFIG_MENDER_PLATFORM_TLS_TYPE="generic/mbedtls" -DCONFIG_MENDER_CLIENT_ADD_ON_INVENTORY=ON -DCONFIG_MENDER_CLIENT_ADD_ON_CONFIGURE=OFF -DCONFIG_MENDER_CLIENT_CONFIGURE_STORAGE=OFF -DCONFIG_MENDER_CLIENT_ADD_ON_TROUBLESHOOT=OFF -DCONFIG_MENDER_FULL_PARSE_ARTIFACT=ON -DCONFIG_MENDER_PROVIDES_DEPENDS=ON
+   ```
+
+2. Build the client:
+   ```
+   cmake --build build --parallel $(nproc --all)
+   ```
+
+### Running the Client
+You can now run and connect the client to e.g. hosted Mender:
+```
+./build/mender-mcu-client.elf --mac_address=$MAC_ADDRESS --device_type=$DEVICE_TYPE --tenant_token=$TENANT_TOKEN --artifact_name=$ARTIFACT_NAME
+```
+
+### Creating an Artifact
+Create an artifact (remember to disable compression):
+```
+./mender-artifact write rootfs-image --compression none --artifact-name <artifact_name> --device-type <device_type> --file <file_name>
+```
+The `device_type` in the artifact has to match the `device_type` used when running the client.
+
+### Deployment
+After creating and uploading the artifact to the server, you should be able to deploy it to the device.
```

Comment:
What are the steps to do this? How do you accept a device to hosted mender? Questions like these does not hurt to answer here. Also some screenshots might be nice. In general, expect that the readers are not as smart as you are ;)

Reply:
@larsewi We can safely assume that any user reading this knows Mender. On our final README we will have links to docs.mender.io and there there is a tutorial on how to do your first deployment, etc.

Specially screenshots will soon get out of date, we should have them only in docs-mender.io